README.md

```diff
@@ -128,9 +211,14 @@ E(fileP).read().then(contents => {
 });
 ```
 
-### HandledPromise constructor
+*{{This portion of the proposal is unclear.  (It's not even entirely obvious
```

Comment:
Duly noted.  I will work on these comments later.  An example of how to implement E would be useful, but first we must introduce the API for the `HandledPromise` constructor.  Maybe not with examples but with Typescript or JSDoc?